corehq/apps/export/static/export/js/models.js

```diff
@@ -834,6 +837,15 @@ hqDefine('export/js/models', [
         return false;
     };
 
+    ExportColumn.prototype.removeDeletedTagFromCaseName = function () {
+      if(this.isCaseName() && this.is_deleted())
+      {
+        this.is_deleted(false);
+        var new_tags = this.tags().filter(ele => ele !== "deleted");
```

Comment:
I might be behind the times, but I think our policy is still not to use full ES6 syntax in our JS code. @orangejenny can you confirm?

Reply:
We don't have an official policy. I'd lean against using ES6 because it will break the linter and because we don't have any plans to switch over, so being consistent right now means using ES5. We don't have any plans to switch the linter.

Reply:
Okay. I have changed the function implementation to ES5.